### PR TITLE
Feature/ci

### PR DESCRIPTION
**Pull request by sdepold**
This is adding releasing with semantic-release

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -87,5 +90,34 @@
   "homepage": "https://github.com/sequelize/cli",
   "engines": {
     "node": ">=10.0.0"
+  },
+  "commitlint": {
+    "extends": [
+      "@commitlint/config-angular"
```
Comment (WikiRik) [inline package.json:96]: Any specific reason why you use this configuration?

File: package.json [hunk 2]
```diff
@@ -87,5 +90,34 @@
...
+          "revert",
+          "style",
+          "test",
+          "meta"
+        ]
+      ]
+    }
+  },
+  "husky": {
+    "hooks": {
```
Comment (WikiRik) [inline package.json:119]: Maybe add a pre-commit hook that runs the linter?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -8,6 +8,7 @@ jobs:
       fail-fast: false
       matrix:
         node-version: [10, 12, 14]
```
Comment (WikiRik) [inline .github/workflows/ci.yml:10]: Node 16 is now LTS, maybe include that as well? Not sure what the policy is from the main repo on supported Node versions

**Review by WikiRik — COMMENTED** [review]
Looks good, just have a few questions/remarks

File: package.json (hunk 2 above)
Comment (WikiRik) [reply package.json:119]: And possibly with lint-staged to make it quicker

**Comment by sdepold** [comment]
@WikiRik I'll have a look into your suggestions today or throughout the weekend!

File: package.json (hunk 1 above)
Comment (sdepold) [reply package.json:96]: it's a copy from sequelize/sequelize

**Pull request merged**